pkg/ocm/client.go

```diff
@@ -118,7 +118,12 @@ func (b *ClientBuilder) Build() (result *Client, err error) {
 	// values in the configuration, so that default values won't be overridden:
 	builder := sdk.NewConnectionBuilder()
 	builder.Logger(logger)
-	builder.Agent(info.UserAgent + "/" + info.Version + " " + sdk.DefaultAgent)
+
+	userAgent := info.DefaultUserAgent
+	if b.cfg.UserAgent != "" {
```

Comment:
shouldn't this be checking if it is empty?

Reply:
I'm setting the `userAgent` variable to the default value, then in line 123 I'm checking if the config has a non-empty UseAgent, I set the `userAgent` value to the provided one instead of the default one.